client/landing/stepper/declarative-flow/internals/steps-repository/get-current-theme-software-sets/index.tsx

```diff
@@ -44,42 +52,69 @@ const GetCurrentThemeSoftwareSets: Step = function GetCurrentBundledPluginsStep(
 	const hasLoadedSiteFeatures: boolean = useSelector( ( state ) =>
 		getHasLoadedSiteFeatures( state, site?.ID || -1 )
 	);
+	const isRequestingActiveTheme: boolean = useSelector( ( state ) =>
+		getIsRequestingActiveTheme( state, site?.ID || -1 )
+	);
```

Comment:
General comment here for this pattern: If we don't have the site yet, should we just assume these values are false?
```suggestion
const hasLoadedSiteFeatures: boolean = ! site?.ID ? false : useSelector( ( state ) =>
		getHasLoadedSiteFeatures( state, site.ID )
	);
	const isRequestingActiveTheme: boolean = ! site?.ID ? false: useSelector( ( state ) =>
		getIsRequestingActiveTheme( state, site.ID )
	);
```
Definitely not blocking, but I am concerned about the case where we don't have the current site, have technically loaded and fetched the details for site ID -1.